zombienet_tests/0002-pov_recovery.toml

```diff
@@ -33,12 +33,12 @@ cumulus_based = true
   validator = true
   image = "{{COL_IMAGE}}"
   command = "test-parachain"
-  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "--use-null-consensus", "--disable-block-announcements"]
+  args = ["--use-null-consensus", "--disable-block-announcements"]
 
   # run eve as parachain full node that is only connected to dave
   [[parachains.collators]]
   name = "eve"
   validator = false
   image = "{{COL_IMAGE}}"
   command = "test-parachain"
-  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "--use-null-consensus", "--disable-block-announcements"]
+  args = ["--reserved-only", "--reserved-nodes {{'dave'|zombie('multiAddress')}}",  "--use-null-consensus", "--disable-block-announcements"]
```

Comment:
We need to pass the `--reserved-node]` flag to the polkadot node, so after the `--`.

Reply:
How did it work before? :)

Reply:
Is zombienet probably adding other nodes as bootnodes automatically? IDK :P